{
  try {
    succeeded(description);
  }
 catch (  Throwable t) {
    errors.add(t);
  }
}
